src/main/java/gov/loc/repository/bagit/writer/BagWriter.java

```diff
@@ -188,22 +188,40 @@ public static void writeTagManifests(final Set<Manifest> tagManifests, final Pat
   }
   
   private static void writeManifests(final Set<Manifest> manifests, final Path outputDir, final Path relativeTo, final String filenameBase, final Charset charsetName) throws IOException{
-    for(final Manifest manifest : manifests){
+     for(final Manifest manifest : manifests){
```

Comment:
The whitespace indentation here is a little jumbled to me – this appears to be 3 spaces and then one space below?